examples/fluids/navierstokes.h

```diff
@@ -33,23 +33,23 @@ typedef enum {
   EULER_TEST_4                 = 4,
   EULER_TEST_5                 = 5,
 } EulerTestType;
-static const char *const EulerTestTypes[] = {"isentropic_vortex", "test_1",      "test_2", "test_3", "test_4", "test_5",
+static const char *const EulerTestTypes[] = {"ISENTROPIC_VORTEX", "TEST_1",      "TEST_2", "TEST_3", "TEST_4", "TEST_5",
                                              "EulerTestType",     "EULER_TEST_", NULL};
```

Comment:
```suggestion
static const char *const EulerTestTypes[] = {"TEST_ISENTROPIC_VORTEX", "TEST_1",      "TEST_2", "TEST_3", "TEST_4", "TEST_5",
                                             "EulerTestType",     "EULER_", NULL};
```
minor, but one way or another these should have a matching pattern